### ISSUE TITLE
Hello this color prop is not possible?  :/

### ISSUE THREAD

**Issue by Zsena** [L-Blondy/tw-colors#9]
example: 

html: 

`<div class="text-mysite/base-primary">TestDiv</div>`

Tailwind config: 

```
plugins: [
        createThemes({
            ro: {
                "mysite/base-primary": "red",
            },
            hu: {
                "mysite/base-primary": "pink",
            },
        }),
    ],
```
I got this error in the rendered page on a style place: 
  
```
   ! .text-mysite\/base-primary {
    --tw-text-opacity: 1;
    color: hsl(var(--twc-mysite/base-primary) / var(--twc-mysite/base-primary-opacity, var(--tw-text-opacity)));
   }
```
   
   Tailwind support this classname format. Can you help me? How can I fix this? Thanks :)

**Comment by L-Blondy** [comment]
Fixed in 1.2.3

**Comment by L-Blondy** [comment]
Hence, fixed in 1.2.4, sorry about that

**Comment by Zsena** [comment]
@L-Blondy thanks for your help. Tomorrow, I will check it. :)
